dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/domain/Account.java

```diff
@@ -44,36 +44,36 @@
    @Nullable
    public abstract String customDefined2();
 
-   public abstract String orgId();
+   @Nullable
+   public abstract String state();
 
    @Nullable
-   public abstract Roles roles();
+   public abstract List<RoleType> roles();
 
-   Account() {
+   @Nullable
+   public abstract AccountPhoneNumber phone();
+
+   public abstract AccountOrganization organization();
+
+   @SerializedNames({ "userName", "fullName", "firstName", "lastName", "emailAddress", "roles", "phone", "department",
+         "customDefined1", "customDefined2", "organization", "state" })
```

Comment:
jclouds makes some assumptions when serializing and deserializing. The order of the parameters in this annotation **must be the same** than the order in which the variables are declared in the class. Please, reorder accordingly.